ambry-network/src/main/java/com.github.ambry.network/Selector.java

```diff
@@ -347,6 +358,24 @@ public void poll(long timeoutMs, List<NetworkSend> sends)
           close(key);
         }
       }
+
+      Iterator<Transmission> sslTransIter = pendingSslHandshakes.iterator();
```

Comment:
Please consider if better to make this part as a method, and call it from `poll()`.